src/pages/workspace/reportFields/WorkspaceAddValuePage.tsx

```diff
@@ -0,0 +1,85 @@
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
+import INPUT_IDS from '@src/types/form/WorkspaceReportFieldsForm';
+
+type WorkspaceAddValuePageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.REPORT_FIELDS_ADD_VALUE>;
+
+function WorkspaceAddValuePage({route}: WorkspaceAddValuePageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const {inputCallbackRef} = useAutoFocusInput();
+    const [formDraft] = useOnyx(ONYXKEYS.FORMS.WORKSPACE_REPORT_FIELDS_FORM_DRAFT);
+
+    const validate = useCallback(
+        (values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_REPORT_FIELDS_FORM>) =>
+            validateReportFieldListValueName(values[INPUT_IDS.VALUE_NAME].trim(), '', formDraft?.[INPUT_IDS.LIST_VALUES] ?? [], INPUT_IDS.VALUE_NAME),
+        [formDraft],
+    );
+
+    const createValue = useCallback((values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_REPORT_FIELDS_FORM>) => {
+        ReportFields.createReportFieldsListValue(values[INPUT_IDS.VALUE_NAME]);
+        Keyboard.dismiss();
+        Navigation.goBack();
+    }, []);
+
+    return (
+        <AccessOrNotFoundWrapper
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+            policyID={route.params.policyID}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_REPORT_FIELDS_ENABLED}
+        >
+            <ScreenWrapper
+                includeSafeAreaPaddingBottom={false}
+                style={styles.defaultModalContainer}
+                testID={WorkspaceAddValuePage.displayName}
+                shouldEnableMaxHeight
+            >
+                <HeaderWithBackButton
+                    title={translate('workspace.reportFields.addValue')}
+                    onBackButtonPress={Navigation.goBack}
+                />
+                <FormProvider
+                    formID={ONYXKEYS.FORMS.WORKSPACE_REPORT_FIELDS_FORM}
+                    onSubmit={createValue}
+                    submitButtonText={translate('common.save')}
+                    validate={validate}
+                    style={[styles.mh5, styles.flex1]}
+                    enabledWhenOffline
+                >
+                    <InputWrapper
+                        InputComponent={TextInput}
+                        maxLength={CONST.WORKSPACE_REPORT_FIELD_POLICY_MAX_LENGTH}
+                        label={translate('common.value')}
+                        accessibilityLabel={translate('common.value')}
+                        inputID={INPUT_IDS.VALUE_NAME}
+                        role={CONST.ROLE.PRESENTATION}
+                        ref={inputCallbackRef}
+                    />
+                </FormProvider>
+            </ScreenWrapper>
+        </AccessOrNotFoundWrapper>
+    );
+}
+
+WorkspaceAddValuePage.displayName = 'WorkspaceAddValuePage';
```

Comment:
I hink we should use a better name, is this for creating the initial value for the report field?

Reply:
Yes, but I am considering reusing this screen for the edit flow. So it is adding a new value to the list.

Reply:
What do you think about **WorkspaceAddListValuePage**?

Reply:
`ReportFieldAddListValuePage` I think will be the best as we are not reusing it for other features than the report fields

Reply:
I've renamed a few components:

1. **WorkspaceAddValuePage** -> **ReportFieldAddListValuePage** (15f33ea84d0354aa96cc99e191c09fe0971074fd)
2. **WorkspaceListValuesPage** -> **ReportFieldListValuesPage** (64fca652a1ff11551c541e589286416388f564a4)
3. **EditValuePage** -> **ReportFieldEditValuePage** (256ca29af11a999f948a8dfa12d55ed6b752b405)

cc @shubham1206agra